pywhat/what.py

```diff
@@ -68,13 +68,41 @@ def parse_options(rarity, include_tags, exclude_tags):
     help="Filter by rarity. Rarity is how unlikely something is to be a false-positive. The higher the number, the more unlikely. This is in the range of 0:1. To filter only items past 0.5, use 0.5: with the colon on the end. Default 0.1:1",
     default="0.1:1",
 )
-@click.option("-i", "--include_tags", help="Only print entries with included tags.")
-@click.option("-e", "--exclude_tags", help="Exclude tags.")
+@click.option("-i", "--include", help="Only print entries with included tags.")
+@click.option("-e", "--exclude", help="Exclude tags.")
 @click.option("-o", "--only-text", is_flag=True, help="Do not scan files or folders.")
 @click.option("-k", "--key", help="Sort by the specified key.")
 @click.option("--reverse", is_flag=True, help="Sort in reverse order.")
+@click.option(
+    "-br",
+    "--boundaryless-rarity",
+    help="Same as --rarity but for boundaryless.",
+    default="0.1:1",
+)
+@click.option(
+    "-bi", "--boundaryless-include", help="Same as --include but for boundaryless."
+)
+@click.option(
+    "-be", "--boundaryless-exclude", help="Same as --exclude but for boundaryless."
+)
+@click.option(
+    "-db", "--disable-boundaryless", is_flag=True, help="Disable boundaryless mode."
+)
 @click.option("--json", is_flag=True, help="Return results in json format.")
-def main(text_input, rarity, include_tags, exclude_tags, only_text, key, reverse, json):
+def main(
+    text_input,
+    rarity,
+    include,
+    exclude,
+    only_text,
+    key,
+    reverse,
+    boundaryless_rarity,
+    boundaryless_include,
+    boundaryless_exclude,
+    disable_boundaryless,
+    json,
+):

```

Comment:
Would we want to switch to `**kwargs`` now? :)

Reply:
I cannot find any info regarding how to do this with click(

Reply:
Just change it to **kwargs, see https://github.com/Ciphey/Ciphey/blob/master/ciphey/ciphey.py#L133